appveyor.yml

```diff
@@ -3,7 +3,7 @@ image: Visual Studio 2019
 
 version: '{build}-{branch}'
 environment:
-  ASSEMBLY_VERSION: 10.2.0
+  ASSEMBLY_VERSION: 10.3.0

```

Comment:
Want a nuget package for:

https://github.com/Brightspace/D2L.Security.OAuth2/commit/4ee069b2a5781dfeba04109e340a23326aa28ead

Reply:
![image](https://user-images.githubusercontent.com/12101120/114192692-ca2c4e00-991b-11eb-8179-d450a540a789.png)